src/baseWrapper.ts

```diff
@@ -115,6 +116,9 @@ export default abstract class BaseWrapper<ElementType extends Node>
   findComponent<T extends ComponentPublicInstance>(
     selector: T | FindComponentSelector
   ): VueWrapper<T>
+  // Find my CatchAll component
+  findComponent<T extends Component>(selector: T): DOMWrapper<Node>
+  findComponent<T extends Component>(selector: string): DOMWrapper<Element>
```

Comment:
This is a more generic way to catch components, I've removed the other `findComponent` overloads and the tests still passed, still not sure if removing the other overloads would cause an issue

Reply:
Shouldn't these methods return a `VueWrapper<T>`?

Reply:
Yes 🤦